frame/elections/src/mock.rs

```diff
@@ -257,7 +257,7 @@ pub(crate) fn balances(who: &u64) -> (u64, u64) {
 }
 
 pub(crate) fn locks(who: &u64) -> Vec<u64> {
-	Balances::locks(who).iter().map(|l| l.amount).collect::<Vec<u64>>()
+	Balances::locks(who).inner().iter().map(|l| l.amount).collect::<Vec<u64>>()
```

Comment:
This can be improved, I should be able to use the `iter` directly defined on `BoundedVec`